pmultiqc/modules/quantms/histogram.py

```diff
@@ -0,0 +1,119 @@
+from collections import OrderedDict, defaultdict
+
+class Histogram:
+
+    def classify(self, plot_cat, with_remains = False, data_cats = None):
+        # with_remains -> Whether there are any remaining values outside the statistical range.
+        # data_cats -> Make statistics and display according to different data categories.
+        # plot_cat = 'frequency' -> Count the number of occurrences of different values.
+        # plot_cat = 'range' -> Count the frequency with which a value occurs over several consecutive ranges.
+        # plot_cat = 'record' -> Assigns values in order according to the different ranges provided.
+
+        self.data = dict()
+        self.data_cats = data_cats
+        if plot_cat == 'frequency':
+            if with_remains:
```

Comment:
I also think you dont need with_remains. Everyting bigger than the last break will always go into the last bin.